Start-MiningPoolHub-Algo.bat

```diff
@@ -11,7 +11,7 @@ if not "%GPU_MAX_ALLOC_PERCENT%"=="100" (setx GPU_MAX_ALLOC_PERCENT 100) > nul
 if not "%GPU_SINGLE_ALLOC_PERCENT%"=="100" (setx GPU_SINGLE_ALLOC_PERCENT 100) > nul
 if not "%CUDA_DEVICE_ORDER%"=="PCI_BUS_ID" (setx CUDA_DEVICE_ORDER PCI_BUS_ID) > nul
 
-set "command=& .\multipoolminer.ps1 -Wallet 1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb -UserName aaronsace -WorkerName multipoolminer -Region europe -Currency btc,usd,eur -DeviceName amd,nvidia,cpu -PoolName miningpoolhub -Donate 24 -Watchdog -MinerStatusURL https://multipoolminer.io/monitor/miner.php -SwitchingPrevention 2"
+set "command=& .\multipoolminer.ps1 -Wallet 1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb -UserName aaronsace -WorkerName multipoolminer -Region europe -Currency btc,usd,eur -DeviceName amd,nvidia,cpu -PoolName miningpoolhub-algo,miningpoolhub-coin -Donate 24 -Watchdog -MinerStatusURL https://multipoolminer.io/monitor/miner.php -SwitchingPrevention 2"
```

Comment:
Is there a bug? These statements are meant to be identical: 
`-PoolName miningpoolhub`
`-PoolName miningpoolhub-algo,miningpoolhub-coin`

Reply:
> Is there a bug?

It seems so, because miningpoolhub is not being expanded to use miningpoolhub-algo & miningpoolhub-coin

The bat doesn't need renaming...
Not if above was working....

Reply:
This scans all configured pools. We need to add the -algo / -coin stuff here.

https://github.com/MultiPoolMiner/MultiPoolMiner/blob/ee6cae556ff68ab572078c9b38f4695ebb5a2352/MultiPoolMiner.ps1#L398
https://github.com/MultiPoolMiner/MultiPoolMiner/blob/ee6cae556ff68ab572078c9b38f4695ebb5a2352/MultiPoolMiner.ps1#L1334

These changes here are probably not necessary:
https://github.com/MultiPoolMiner/MultiPoolMiner/commit/ee6cae556ff68ab572078c9b38f4695ebb5a2352

Reply:
Thanks for the info; I'll sort it our shortly.

Reply:
That turned into a big task... think it's all sorted...